src/compiler/scanner.ts

```diff
@@ -2117,8 +2122,9 @@ namespace ts {
             }
 
             if (isIdentifierStart(ch, ScriptTarget.Latest)) {
-                while (isIdentifierPart(text.charCodeAt(pos), ScriptTarget.Latest) && pos < end) {
-                    pos++;
+                let nextChar: number;
+                while (isIdentifierPart(nextChar = codePointAt(text, pos), ScriptTarget.Latest) && pos < end) {

```

Comment:
is it intentional not to allow unicode escape sequences in JSDoc identifiers?

Reply:
I believe so, but I dunno. Given that it's a comment, it just means that we'll interpret the `\u0061`  verbatim. @sandersn would be the authority here.

Reply:
I'm confused. Doesn't this change cause us to allow escape sequences in JSDoc identifiers?

jsdoc.app doesn't mention unicode as far as google can tell, so we are free to do whatever we want. I vote that JSDoc identifiers work the same as normal ones, which should mean allowing unicode escape sequences.